CIP-_/README.md

```diff
@@ -0,0 +1,210 @@
+---
+CIP: ?
+Title: Optimized Coin Selection Algorithms for Cardano
+Category: Wallets
+Authors:
+    - Nicholas Maselli <maselli.nicholas@gmail.com>
+Implementors: CardanoSharp 7, SaturnNFT, Levvy Finance
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/785
+Status: Proposed
+Created: 2024-03-14
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Coin Selection algorithms have become a critical part of the Cardano ecosystem. These algorithms choose a set of input Utxos for a given set of new Outputs.
+
+This CIP aims to present optimized coin selection algorithms to enable Wallets and Dapps to interact with smart contracts and native assets such as tokens and NFTs in a more efficient way. The goal of these algorithms is to primarily reduce the size of transactions as well as reduce the mem and step required for smart contract executions.
+
+## Motivation: why is this CIP necessary?
+
+<b>Cardano needs to scale.</b> As the ecosystem grows and new users come in, blocks are regular at 100% load with 88KB per block allocation. Recently, events such as the Lil Sappys NFT mint have pushed the Cardano blockchain load to over 100% for several hours, preventing anyone from using the chain and providing an awful user experience to new users who complain on social media and give the ecosystem a bad reputation.
+
+<b>This can't continue if Cardano is to succeed.</b>
+
+Currently the existing <a href="https://github.com/cardano-foundation/CIPs/tree/master/CIP-0002">CIP-2</a> for coin selection is not sufficient as it was developed in a time before native assets and smart contracts were live on the Cardano Blockchain. Many existing dapps are using extremely poor coin selection resulting in transactions that are 2,3, sometimes 5 times larger then neccesary while also creating outputs that are difficult to handle in future coin selections.
```

Comment:
> Many existing dapps are using extremely poor coin selection resulting in transactions that are 2,3, sometimes 5 times larger then neccesary while also creating outputs that are difficult to handle in future coin selections.

Are you able to include some worked examples of transactions that are 2, 3, and sometimes 5 times larger than necessary?